src/v2/view-builder/views/ov/ChallengeOktaVerifyResendPushView.js

```diff
@@ -3,6 +3,7 @@ import { BaseForm } from '../../internals';
 import BaseAuthenticatorView from '../../components/BaseAuthenticatorView';
 import AuthenticatorVerifyFooter from '../../components/AuthenticatorVerifyFooter';
 
+const OV_NMC_FORCE_UPGRAGE_KEY = 'idx.authenticator.app.method.push.force.upgrade.number_challenge';
```

Comment:
If this is going to be a pattern we follow for other error messages, consider moving this to `Constants.js` or `AuthenticatorUtil.js` as an object with the title mapping. For example:

```js
const ERRORS_WITH_TITLES = {
  'idx.authenticator.app.method.push.force.upgrade.number_challenge': {
    titleKey: 'idx.authenticator.app.method.push.force.upgrade.number_challenge.title'
  }
}
```

Reply:
Good callout. I was going to do that in my next PR https://oktainc.atlassian.net/browse/OKTA-375573